File: apis/src/main/java/io/dirigible/samples/api/domain/input/ExampleRequest.java
package io.dirigible.samples.api.domain.input;

public class ExampleRequest {

	private String exampleId;
	private String exampleName;

	public String getExampleId() {
		return exampleId;
	}

	public void setExampleId(String exampleId) {
		this.exampleId = exampleId;
	}

	public String getExampleName() {
		return exampleName;
	}

	public void setExampleName(String exampleName) {
		this.exampleName = exampleName;
	}

	public ExampleRequest withExampleId(String exampleId) {
		setExampleId(exampleId);
		return this;
	}

	public ExampleRequest withExampleName(String exampleName) {
		setExampleName(exampleName);
		return this;
	}
}

File: apis/src/main/resources/META-INF/dirigible/custom-api/ExampleRequest.ts
export class ExampleRequest {

    // @ts-ignore
    private exampleId: string;

    // @ts-ignore
    private exampleName: string;

    public getExampleId(): string {
        return this.exampleId;
    }

    public setExampleId(exampleId: string): void {
        this.exampleId = exampleId;
    }

    public getExampleName(): string {
        return this.exampleName;
    }

    public setExampleName(exampleName: string): void {
        this.exampleName = exampleName;
    }

    public withExampleId(exampleId: string): ExampleRequest {
        this.setExampleId(exampleId);
        return this;
    }

    public withExampleName(exampleName: string): ExampleRequest {
        this.setExampleName(exampleName);
        return this;
    }
}